activiti-engine/src/test/java/org/activiti/engine/test/api/event/DatabaseEventLoggerTest.java

```diff
@@ -521,11 +521,19 @@ public void testStandaloneTaskEvents() throws JsonParseException, JsonMappingExc
     task.setAssignee("kermit");
     task.setTenantId("myTenant");
     taskService.saveTask(task);
+    
+    taskService.setAssignee(task.getId(), "gonzo");
+    
+    task = taskService.createTaskQuery().taskId(task.getId()).singleResult();
+    task.setAssignee("kermit");
+    taskService.saveTask(task);
 
     List<EventLogEntry> events = managementService.getEventLogEntries(null, null);
-    assertEquals(2, events.size());
-    assertEquals("TASK_ASSIGNED", events.get(0).getType());
-    assertEquals("TASK_CREATED", events.get(1).getType());
+    assertEquals(4, events.size());
+    assertEquals("TASK_CREATED", events.get(0).getType());
+    assertEquals("TASK_ASSIGNED", events.get(1).getType());
```

Comment:
@balsarori why so many??

Reply:
Each time assignee changes an event should be triggered. Here I want to make sure that all cases are covered and that my changes didn't miss a case :).